src/onLayout/identifyControls.js

```diff
@@ -0,0 +1,5 @@
+export default function identifyControls() {
```

Comment:
We should probably make an issue to fix this in webcharts.

Reply:
It would be nice to identify controls without accessing their data.